src/main/java/com/google/api/codegen/config/SampleSpec.java

```diff
@@ -36,10 +34,10 @@
   /** A reference to the Sample Configuration from which the other fields are derived. */
   private final SampleConfiguration sampleConfiguration;
 
-  /** All the SampleValueSets defined for this method, indexed by their IDs. */
-  private Map<String, SampleValueSet> valueSets;
+  /** All the SampleValueSets defined for this method. */
+  private final List<SampleValueSet> valueSets;
```

Comment:
OK, makes sense given the accessors we have. I was thinking we were going to need random access by ID, but we don't yet. We can worry about it when we do.

Reply:
SG